templates/zerver/api/get-messages.md

```diff
@@ -64,14 +64,13 @@ zulip(config).then((client) => {
 {tab|curl}
 
 ``` curl
-curl -X GET {{ api_url }}/v1/messages \
-    -u BOT_EMAIL_ADDRESS:BOT_API_KEY \
-    -d "anchor=42" \
-    -d "use_first_unread_anchor=false" \
-    -d "num_before=3" \
-    -d "num_after=14" \
-    -d 'narrow=[{"operator":"stream", "operand":"party"}]' \
-
+curl -X GET -G http://localhost:9991/api/v1/messages \
+    -u outgoing-webhook@zulip.com:MMSm1bGyN1DW9OFQH4IeZ7zTgsiLL2oX \
+    -d num_before=3 \
+    -d num_after=14 \
+    -d anchor=42 \
+    -d use_first_unread_anchor=false \
```

Comment:
We might want to use `'`s around these as part of our standards, to make clear the right way to quote things.  I guess integers/booleans will never have spaces in them, but we might want to do that for strings.

Reply:
I've added them as `"`s based on the what I saw in the example before I changed it.